Source/Extensions/Blazorise.DataGrid/_DataGridRowMultiSelect.razor

```diff
@@ -1,6 +1,6 @@
 ﻿@typeparam TItem
 @inherits _BaseDataGridRowMultiSelect<TItem>
-<TableRowCell @onclick:preventDefault Class="@(Column.CellClass?.Invoke(Item))" Style="@BuildCellStyle()" TextAlignment="@Column.TextAlignment" VerticalAlignment="@Column.VerticalAlignment" Display="@Column.Display" >
+<TableRowCell @onclick:preventDefault="!IsBS5Provider()" Class="@(Column.CellClass?.Invoke(Item))" Style="@BuildCellStyle()" TextAlignment="@Column.TextAlignment" VerticalAlignment="@Column.VerticalAlignment" Display="@Column.Display" >
```

Comment:
I don't like doing it like this, naming or being tied to any specific provider. Might be better to add a new "provider" class with behavioral properties, or just add another method into ClassProvider, eg. `TableRowClickPrevent() => true`. Something like that.

Reply:
Well yea, you are quite right

Reply:
Altough the naming in this case should be something along the lines of DataGridMultiSelectClickPrevent or smthg, as TableRowClickPrevent is too general, I guess.

Reply:
Do you know why exactly BS5 needs to have prevented click disabled on a row?

Reply:
I can reinvestigate certainly has to do with how they handle the check

Reply:
https://github.com/Megabit/Blazorise/pull/3028